lib/exceptions.js

```diff
@@ -0,0 +1,36 @@
+var records = require('./exceptions.json');
+
+function compareValue(data, ref) {
+    return (ref === undefined) || (data === ref);
+}
+
+function findSet(shortname) {
+    var count = 0;
+    function _findSet(name) {
+        if (count > 10)
+           return undefined;
+        var set = records[name];
+        count++;
+        if (typeof set === "String")
+            return _findSet(set);
+        return set;
+    }
+    return _findSet(shortname);
+}
```

Comment:
Sorry, as `exceptions.json` is empty and there are no examples or test cases provided, I'm finding it difficult to understand the recursion here, and that limit depth of 10&hellip; What is that for?

This is what I entered in my `exceptions.json` file to try to mute the only error I get when [checking locally a WD that is already published](http://localhost:3000/?url=http%3A%2F%2Fwww.w3.org%2FTR%2Fencrypted-media%2F&profile=WD-Echidna&validation=simple-validation&noRecTrack=false&informativeOnly=false&echidnaReady=true&patentPolicy=pp2004&processDocument=2015):

```json
{
  "encrypted-media": [
    {"rule": "echidna.todays-date"}
  ]
 }
```

Is that valid syntax?